package.json

```diff
@@ -3,6 +3,7 @@
 	"version": "2.8.1",
 	"scripts": {
 		"start": "cross-env NODE_ENV=development CALYPSO_CLIENT=true webpack-dev-server --hot --inline --watch --content-base dist --port 8085 --host 0.0.0.0",
+		"watch": "cross-env NODE_ENV=development CALYPSO_CLIENT=true webpack --watch",
```

Comment:
I needed this for my local dev. 😅 Can be removed.

Reply:
No way! Keeper for sure! 😄